MIGRATING-0.6-0.7.md

```diff
@@ -161,7 +160,58 @@ Driver authors should use `DrawTarget` exported by the [`embedded-graphics-core`
 
 `DrawTarget` now uses an associated type for the target color instead of a type parameter.
 
-`DrawTarget`s must also implement the `Dimensions` trait.
+`DrawTarget`s must also implement the `OriginDimensions` trait.
```

Comment:
`DrawTarget`s aren't required to implement `OriginDimensions`. But it is recommended to implement `OriginDimensions` for most display drivers, because the drawable area starts in the origin.

Reply:
Good clarification, thanks. I added some text about the `Dimensions` trait bound and suggesting `OriginDimensions` instead. Let me know what you think - maybe it's superfluous and we should just mention `OriginDimensions`?